src/data/markdown/docs/03 cloud/05 Project and Team Management/05 SAML SSO.md

```diff
@@ -41,46 +47,135 @@ To setup SAML SSO for your organization, some requirements should be satisfied a
+| Signature Algorithm        | RSA_SHA256                             |
+| Digest Algorithm           | SHA256                                 |
+| Assertion Encryption       | Encrypted                              |
+| SAML Single Logout         | Disabled                               |
+| authnContextClassRef       | PasswordProtectedTransport             |
+| Honor Force Authentication | Yes                                    |
+| SAML Issuer ID             | http://www.okta.com/${org.externalKey} |
 
-## Assigning default project(s) to newly provisioned users
+### Attribute Statements
 
-When new users are provisioned into k6 Cloud, they automatically join your organization and will be assigned the default project of your organization. You can change these default settings and choose which projects to assign to newly provisioned users.
+These are required.
 
-It is recommended to update the list of projects before any new users are provisioned, so that you can assign them to pre-defined projects. Please note that this feature doesn't work retroactively, that is, already existing (provisioned) users on k6 Cloud won't be affected by this change.
+| Name            | Name Format | Value              |
+| --------------- | ----------- | ------------------ |
+| user.email      | Basic       | user.email         |
+| user.username   | Basic       | user.login         |
+| user.first_name | Basic       | user.firstName     |
+| user.last_name  | Basic       | user.lastName      |
+| token           | Basic       | \<TOKEN-YOU-GET-FROM-K6-SUPPORT\> |
 
-Follow these steps to update list of default projects for new users:
+### Group Attribute Statements
 
-1. Make sure you correctly setup the SAML organization with the help of the customer success team. You can use your admin account to test the setup.
-2. Go to your organization setting and choose the SAML SSO tab:
-![SAML SSO settings](images/04-SAML-SSO/saml-sso-settings.png)
-As you can see below, the default project is already selected:
-![SAML SSO default project](images/04-SAML-SSO/saml-sso-default-project.png)
-3. From the drop-down you can choose the default project(s) newly provisioned users will be assigned, then click on update button:
-![SAML SSO update default projects](images/04-SAML-SSO/saml-sso-update-default-project.png)
-4. Now the users assigned to the IdP application can sign in to k6 Cloud and they'll be automatically added to the specified project(s) in step 3 with __project read/write__ permissions.
+This is optional. If not given, all users will end up starting in the k6 Organization's _default_ Project(s) or Team(s) (configurable from the app, see next section).
+
+Setting Okta up to send Group attributes, allows for more powerful mapping of Okta Groups to k6 cloud Teams/Projects. Whatever you choose to add here, you must communicate it to the k6 customer success team so they can set it the matching mapping on the k6 cloud side.
```

Comment:
Should we capitalize `Group` here?

Reply:
I felt this made it clearer that a Group is a special entity on the IdP side and not just a generic group of people.